utils/build/docker/golang/parametric/install_ddtrace.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
There should not have any difference for parametric and end-to-end scenarios at this point. Meaning if  there is something not working in the current script, it should affect both, and must be corrected.

Could you precise what's not working with the current script ?

Reply:
Yes.
[This PR](https://github.com/DataDog/system-tests/commit/eff4b2bb85aec0110b964d9626e2fb803543c4b6) presented a change to the binaries documentation that added new instructions for specifying dd-trace-go build in parametric apps, see `Golang `section. (Note: I don't have context on the motivation for this PR).

This info introduced in that PR work for parametric apps, but the way for weblog apps is using the `binaries/golang-from-go-get` method (which was for some reason erased entirely in that PR, but I added it back in in [this PR](https://github.com/DataDog/system-tests/commit/5f596804ee1055a1d71dfc7dfec8ab787198c007)). binaries.md currently has two sets of instructions: one for weblog and one for parametric.

After some more testing, it became clear that the parametric instructions don't work because of the install_ddtrace.sh script, which would ignore the dd-trace-go version specified in the go.mod file. Therefore, I updated the install script for the parametric app to use the dd-trace-go version specified in the go.mod file, to conform to the parametric instructions.

**Current state (with this PR)**: 
- specify dd-trace-go version in parametric app with `go get` inside of /golang/parametric
- specify version in weblog app with `binaries/golang-from-go-get`

**Do you prefer just a single way to specify dd-trace-go version instead?, e.g**,:
- specify version with `binaries/golang-from-go-get` for both parametric and weblog

If so^, then I would delete this PR and just update the binaries.md instructions.

Reply:
> Do you prefer just a single way to specify dd-trace-go version instead?, e.g,:

Yes, definitely. We must have a unique script for both scenario, and it should works for all described use cases.